### PR TITLE
Added step 5

### PR DESCRIPTION
**Pull request by ghost**
Notes:
-The loop reading the story to a string reads in the first 128 unicode characters (so the loop will stop reading from the file if it encounters a character with a value outside of 0 to 127.)

-variable names need to be adjusted for each list of words

### CODE REVIEW

**Comment by emerygenga** [comment]
+1

File: MadLibs.java [hunk 1]
```diff
@@ -18,21 +18,63 @@ public static void main(String[] args) {
...
+		catch(IOException ex){
+			System.out.println ("Something went wrong. :(");
+		}
+		
+
+
+		//update the variable names for each list!!!!!!! *****
+		//check to see if there are any words that need to be added and add them when needed, then rinse and repeat for every other type of word
+		//nouns (listOfNouns)
+		while (story.indexOf("<<NOUN>>") > -1){
```
Comment (AKHW123) [inline MadLibs.java:57]: Won't this just keep finding the same noun an infinite amount of times?

Comment (AKHW123) [reply MadLibs.java:57]: nvm i see what you're doing

**Comment by ghost** [comment]
I'm going to close this pull request for now and submit an updated pull request.

**Pull request closed without merging**